aepp/flowservice.py

```diff
@@ -514,8 +514,8 @@ def createFlow(
             }
             if schedule_start_time is not None:
                 obj["scheduleParams"]["startTime"] = schedule_start_time
-            if schedule_frequency is not None:
-                obj["scheduleParams"]["frequency"] = schedule_frequency
+            if schedule_time_unit is not None:
+                obj["scheduleParams"]["timeUnit"] = schedule_time_unit
```

Comment:
I don't think that's correct, it seems that depending on whether the flow or a source the field is different, sees to be `frequency` for sources but `timeUnit` for destinations. 

Currently it's just 1 functoin for creating both flows, so maybe we need a `createSourceFlow` and `createDestinationFlow` what do you think?